ifaddr/_shared.py

```diff
@@ -69,7 +75,13 @@ class IP(object):
     Represents an IP address of an adapter.
     """
 
-    def __init__(self, ip, network_prefix, nice_name):
+    # Type of an IPv4 address (a string in "xxx.xxx.xxx.xxx" format)
+    IPv4Address = Union[str]
+
+    # Type of an IPv6 address (a three-tuple `(ip, flowinfo, scope_id)`)
+    IPv6Address = Union[Tuple[str, int, int]]
+
```

Comment:
`Unions` with single items aren't making much sense, just use `str` and `Tuple[str, int, int]` above.

Reply:
This is actually correct however it will be unnecessary when moved outside the class.